test/testResourceString.js

```diff
@@ -1243,5 +1244,118 @@ module.exports.resourcestring = {
 
         test.done();
     },
+    testSourceContextResourceStringStaticHashKey: function(test) {
+        test.expect(1);
+
+        test.equal(SourceContextResourceString.hashKey("qmlapp", "foobar", "de-DE", "This is a test", "x-qml", "flavor"), "scrs_qmlapp_foobar_de-DE_This is a test_x-qml_flavor_");
```

Comment:
Should test here with the sourcehash parameter

Reply:
I updated it